### ISSUE TITLE
Error if build status is NOT_BUILT

### ISSUE THREAD

**Issue by fkrauthan** [RisingOak/jenkins-client#42]
I can't call build.details() if the build status is NOT_BUILT:

```
 Causing: org.quartz.JobExecutionException: org.codehaus.jackson.map.JsonMappingException: Can not construct instance of com.offbytwo.jenkins.model.BuildResult from String value 'NOT_BUILT': value not one of declared Enum instance names
!  at [Source: org.apache.http.conn.EofSensorInputStream@27205751; line: 1, column: 3427] (through reference chain: com.offbytwo.jenkins.model.BuildWithDetails["result"])
```

**Comment by gmaslowski** [comment]
https://github.com/RisingOak/jenkins-client/pull/49

**Comment by khmarbaise** [comment]
Has been fixed with 0.3.0...
